currencyfair.yaml

```diff
@@ -343,8 +343,10 @@ paths:
                   summary: Successful authorization
                   value:
                     $ref: "resources/examples/oauth/oAuthResponse.json"
-        '401':
+        '400':
```

Comment:
400? How does this work, I'm pretty sure that's 401 just like the MFA one?

Reply:
A missing `authorisation_code` parameter in the `oauth` request will return a 400